integration/util.py

```diff
@@ -199,10 +199,24 @@ def created(_):
     return d
 
 
-def await_file_contents(path, contents, timeout=15):
+def await_file_contents(path, contents, timeout=15, error_if=None):
+    """
+    wait up to `timeout` seconds for the file at `path` to have the
+    exact content `contents.
+
+    :param error_if: if specified, a list of additional paths; if any
+        of these paths appear an Exception is raised.
+    """
     start_time = time.time()
     while time.time() - start_time < timeout:
         print("  waiting for '{}'".format(path))
+        if error_if and any([exists(p) for p in error_if]):
```

Comment:
Okay, reasonable.  If Tahoe-LAFS has reproducable builds as a goal (which seems cool to me, though I don't know how many hurdles a Python project has to overcome in general to achieve that, is it plausible?) then maybe encoding this rule somewhere would make sense (in the lint builder?).

Reply:
I don't know that Tahoe does even have that as a goal -- my experience with all that was watching Debian people try to package txtorcon to make it reproducible ;)

I'll see if the linters can do this intelligently (e.g. setting a default to a string or int works fine AFAIK probably 'because immutable')